ChangeLog

```diff
@@ -1514,7 +1514,7 @@
 
         * inst/include/Rcpp/traits/is_na.h: Speed up Rcpp::is_na<double>()
 
-2018-01-09  Iñaki Ucar  <i.ucar86@gmail.com>
+2018-01-09  Iñaki Ucar  <iucar@fedoraproject.org>
```

Comment:
Rewriting old entries smells a little like rewriting history...

Reply:
I did a `sed` in place, didn't look at the diff. Reverted now.